ReactSkia/sdk/WindowDelegator.cpp

```diff
@@ -30,19 +30,20 @@ void WindowDelegator::createWindow(SkSize windowSize,std::function<void ()> wind
 
 void  WindowDelegator::createNativeWindow() {
 
-  displayPlatForm_=RnsShell::PlatformDisplay::sharedDisplayForCompositing().type();
+  displayPlatForm_ = RnsShell::PlatformDisplay::sharedDisplayForCompositing().type();
```

Comment:
WindowDelegator.cpp has only changes w.r.t spacing. Is that intentional changes, if not can you please do revert

Reply:
this is an intentional change.